lib/modules/datasource/artifactory/index.ts

```diff
@@ -114,6 +115,8 @@ export class ArtifactoryDatasource extends Datasource {
   }
 
   private static parseReleaseTimestamp(rawText: string): string {
-    return rawText.trim().replace(regEx(/ ?-$/), '') + 'Z';
+    return (
+      rawText.split(regEx(/\s{2,}/)).filter(e => DateTime.fromFormat(e, "dd-LLL-yyyy HH:mm").isValid)[0] + 'Z'
```

Comment:
@viceice I would have preferred not to make this datetime refactor as part of this fix